### PR TITLE
docs/Installation: fix Apple Developer account link

### PR DESCRIPTION
**Pull request by gerlero**
- [X] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [X] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [ ] Have you successfully run `brew style` with your changes locally?
- [ ] Have you successfully run `brew typecheck` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?

-----

Fixes the link to the Apple Developer account page in the footnote, which is now giving a 404.

Not sure if the "Mac OS X" mention on the same line shouldn't also be changed to "macOS"...

### CODE REVIEW

**Review by carlocab — APPROVED** [review]
Thanks!

> Not sure if the "Mac OS X" mention on the same line shouldn't also be changed to "macOS"...

If they refer to versions that are called macOS[^1], it should be changed to "macOS".

[^1]: I think this is Sierra and newer?

**Comment by gerlero** [comment]
Well, the section that has that footnote also says macOS Big Sur or higher is required, so "macOS" it is. I'll add that change here too.

> 1. I think this is Sierra and newer? [↩](#user-content-fnref-1-08b0bb28c789258cf3b78d0b3070d36c)

That's right!

File: docs/Installation.md [hunk 1]
```diff
@@ -90,6 +90,6 @@ Uninstallation is documented in the [FAQ](FAQ.md).
 
 <a name="2"><sup>2</sup></a> macOS 11 (Big Sur) or higher is best and supported, 10.11 (El Capitan) – 10.15 (Catalina) are unsupported but may work and 10.10 (Yosemite) and older will not run Homebrew at all. For 10.4 (Tiger) – 10.6 (Snow Leopard) see [Tigerbrew](https://github.com/mistydemeo/tigerbrew).
 
-<a name="3"><sup>3</sup></a> You may need to install Xcode, the CLT, or both depending on the formula, to install a bottle (binary package) which is the only supported configuration. Downloading Xcode may require an Apple Developer account on older versions of Mac OS X. Sign up for free at [Apple's website](https://developer.apple.com/register/index.action).
+<a name="3"><sup>3</sup></a> You may need to install Xcode, the CLT, or both depending on the formula, to install a bottle (binary package) which is the only supported configuration. Downloading Xcode may require an Apple Developer account on older versions of macOS. Sign up for free at [Apple's website](https://developer.apple.com/account/).
```
Comment (MikeMcQuaid) [inline docs/Installation.md:93]: @gerlero @carlocab sorry, this is wrong: it should be `Mac OS X` because it only applies there!

Comment (MikeMcQuaid) [reply docs/Installation.md:93]: https://github.com/Homebrew/brew/pull/15700

Comment (gerlero) [reply docs/Installation.md:93]: Well, I got it wrong then (honestly, I'm not really sure how far back one can go before Homebrew doesn't install anymore). Thanks for the check!

**Pull request merged**